src/main/scala/nl.knaw.dans.easy.properties/app/repository/sql/package.scala

```diff
@@ -28,14 +28,31 @@ package object sql {
 
   val dateTimeFormatter: DateTimeFormatter = ISODateTimeFormat.dateTime()
   val timeZone: DateTimeZone = DateTimeZone.UTC
+
   implicit def timeZoneToCalendar(timeZone: DateTimeZone): Calendar = {
     Calendar.getInstance(timeZone.toTimeZone)
   }
+
   implicit def dateTimeToTimestamp(dt: DateTime): Timestamp = new Timestamp(dt.getMillis)
 
   type PrepStatementResolver = (PreparedStatement, Int) => Unit
+
   def setString(s: String): PrepStatementResolver = (ps, i) => ps.setString(i, s)
+
   def setDepositId(depositId: DepositId): PrepStatementResolver = setString(depositId.toString)
+
   def setInt(int: => Int): PrepStatementResolver = (ps, i) => ps.setInt(i, int)
+
   def setInt(s: String): PrepStatementResolver = setInt(s.toInt)
+
+  implicit class RichPreparedStatement(val preparedStatement: PreparedStatement) extends AnyVal {
+
+    /** @return rowCount */
+    def executeUpdateWith(values: Seq[String]): Int = {
```

Comment:
if we make this a `values: Seq[Any]`, we can use this for more cases in the `foreach` such that we can use `setString`, `setBoolean`, etc.

Reply:
A quick attempt broke tests: https://github.com/DANS-KNAW-jp/easy-deposit-properties/commit/4c5f5f0000538bee2dbb682d65245dc4632eaf98